central/policycategoryedge/datastore/singleton.go

```diff
@@ -21,7 +20,7 @@ func initialize() {
 	var storage store.Store
```

Comment:
nit: we don't need to declare these outside the if anymore since that is the only time we use them.  It might be cleaner now to do

```
if !env.PostgresDatastoreEnabled.BooleanSetting() {
   return nil
}
```

Reply:
Yes, done.